backend/src/test/java/com/htecgroup/skynest/utils/BucketBasicUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.htecgroup.skynest.utils;
+
+import com.htecgroup.skynest.model.entity.CompanyEntity;
+
+import java.time.LocalDateTime;
+
+public class BucketBasicUtil {
+  protected static final LocalDateTime currentDateTime = LocalDateTime.now();
```

Comment:
I would not extract current date in a util class since that is created once at the moment of creation and not when you actually call it..

On the other hand, util classes can be final and with private constructor (I also believe that we have that already in the code)